tools/cli/pkg/ers/client/client.go

```diff
@@ -0,0 +1,54 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/control-plane/tools/cli/pkg/ers"
+	"github.com/pkg/errors"
+)
+
+const environmentsPath = "%s/provisioning/v1/kyma/environments"
+const brokersPath = "%s/provisioning/v1/brokers"
+const pagedParams = "page=%d&size=%d"
+const idParam = "id=%s"
+
+type ersClient struct {
+	url    string
+	client *HTTPClient
+}
+
+func NewErsClient(url string, client *HTTPClient) Client {
+	return &ersClient{
+		url,
+		client,
+	}
+}
+
+func (c *ersClient) GetOne(instanceID string) (*ers.Instance, error) {
+	instances, err := c.client.get(fmt.Sprintf(environmentsPath+"?"+idParam, c.url, instanceID))
```

Comment:
idParam - one parameter, but you are providing two

Reply:
It just a format string. Params include URL and id.